src/games/strategy/triplea/ai/proAI/util/ProUtils.java

```diff
@@ -46,6 +47,26 @@ public ProUtils(final ProAI ai) {
     return unitTerritoryMap;
   }
 
+  public List<PlayerID> getEnemyPlayersInTurnOrder(final PlayerID player) {
```

Comment:
I would say "almost" very testable method. The issue is this "final GameData data = ai.getGameData();". So you need to setup the full AI with initialized game data to test this. This is the big issue right now as I have essentially everything extending from ProAI main class which holds the current game data. This is beneficial in that it doesn't need passed around as parameters but the negative is that they can't be static and are all tied back to that class.

I also ended up completed refactoring this method as well since I needed to create methods to get allied players and all players in turn order.